### ISSUE TITLE
Is XPDF an alternative of poppler tools?

### ISSUE THREAD

**Issue by zuphilip** [UB-Mannheim/zotero-ocr#25]
See also https://github.com/zotero/cross-xpdf

**Comment by kba** [comment]
poppler started as a fork of XPDF, is part of freedesktop and is frequently updated, so I see no reason why the XPDF implementation of pdftotext or pdfinfo would be superior.

**Comment by zuphilip** [comment]
Thank you @kba for the explanation. But then both might work the same for a user, right?

I was just thinking to mention somewhere that instead of poppler also xpdf could be used. It is an prerequisite, i.e. not our main focus, so 2-3 words in the README could enough.

**Comment by kba** [comment]
Yes, both will work but at least on my Ubuntu 18.04 machine, the `xpdf` package does not include the tools, you need to download/install them from https://www.xpdfreader.com/download.html or build them yourself. `poppler-utils` contains the tools.

**Comment by stweil** [comment]
Let us close this really old issue (please re-open if you think it is still useful).
